wd_helper.au3

```diff
@@ -984,8 +984,9 @@ EndFunc   ;==>_WD_ElementOptionSelect
 ; Parameters ....: $sSession       - Session ID from _WD_CreateSession
 ;                  $sSelectElement - Element ID of Select element from _WD_FindElement
 ;                  $sCommand       - Action to be performed. Can be one of the following:
-;                  |OPTIONS - Retrieve array containing value / label attributes from the Select element's options
-;                  |VALUE   - Retrieve current value
+;                  |OPTIONS - Retrieve array containing value / label attributes from the Select element's Options
+;                  |SELECTEDINDEX - Retrieve 1-based index of selected option, from Options collection of designated Select element
```

Comment:
This is wrong. The index is 0-based. You can use -1 to indicate no selection. Please fix.

Reply:
Using this example:
https://www.autoitscript.com/forum/topic/201537-webdriver-example-scripts-collection/?do=findComment&comment=1496107

I have such result:
![image](https://user-images.githubusercontent.com/11089482/152381795-c3fe15a8-b4b8-4852-ab92-46d52c85b791.png)

But As to your request I start checking deeply.

Reply:
Your demo is wrong (was just about to post that here). I think that you are looking at it incorrectly. "Choose option" is a valid choice that is in index position 0.

https://www.w3schools.com/jsref/prop_select_selectedindex.asp

Reply:
After our PM in Au3Forum: 
Can we treat this as resolved conversation ?